polkadot/runtime/parachains/src/hrmp.rs

```diff
@@ -836,6 +836,34 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// TODO write docs
+		#[pallet::call_index(10)]
+		#[pallet::weight(<T as Config>::WeightInfo::establish_system_channel())] // TODO benchmarks
+		pub fn establish_channel_with_system(
+			origin: OriginFor<T>,
+			recipient: ParaId,
+		) -> DispatchResultWithPostInfo {
+			let sender = ensure_parachain(<T as Config>::RuntimeOrigin::from(origin))?;
+
+			ensure!(recipient.is_system(), Error::<T>::ChannelCreationNotAuthorized);
+
+			let config = <configuration::Pallet<T>>::config();
+			let max_message_size = config.hrmp_channel_max_message_size;
+			let max_capacity = config.hrmp_channel_max_capacity;
+
+			Self::init_open_channel(sender, recipient, max_capacity, max_message_size)?;
+			Self::accept_open_channel(recipient, sender)?;
```

Comment:
Most chains would want the system to be able to send them messages too, right? But not want to do a referendum to open a channel. So this should repeat with the reverse.